src/v4l2video/v4l2src.c

```diff
@@ -762,19 +764,39 @@ static void gst_imx_v4l2src_set_property(GObject *object, guint prop_id,
 			break;
 
 		case IMX_V4L2SRC_BRIGHTNESS:
-			v4l2src->brightness = g_value_get_int(value);
+			if (v4l2src->fd_obj_v4l) {
```

Comment:
Should there be an `else` to the `if` statements when `v4l2src->fd_obj_v4l` is NULL?  Same for all the settings.  Should there be an error or at least a message recorded?

Reply:
Added else statement with error message as suggested.